Add a new Vault known issue around rewrapping with multi-seal enabled on failovers (#1000)
hashicorp/web-unified-docs: content/vault/global/partials/important-changes/known-issue/multi-seal-rewrap.mdx

### Full seal rewraps occur on DR/PR failover with multi-seal enabled ((#multi-seal-rewrap)) <EnterpriseAlert inline="true" />

| Change      | Affected versions                              | Fixed version |
|-------------|------------------------------------------------|---------------|
| Known issue | 1.20.x+ent, 1.19.x+ent, 1.18.x+ent, 1.16.x+ent | None          |

> When failing over to a DR or performance cluster with `enable_multiseal = true`, a full rewrap occurs. This
> can lead to performance degradation until the rewrap operation completes.

Reviewer: ```suggestion
A full rewrap happens when Vault fails over to a DR or performance cluster with `enable_multiseal = true`.
The rewrap can lead to performance degradation until the rewrap operation completes.
```
Style correction: write in active voice, avoid "this" as a pronoun

#### Recommendation

> No workaround is available at this time, short of disabling multi-seal support.

Reviewer: ```suggestion
The only workaround is to disable multi-seal support.
```
Style correction: use direct language
